client/src/features/users/page.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { DataGrid } from '@mui/x-data-grid';
+import { Button, Container, Typography, Grid } from '@mui/material';
+import { Edit, Delete } from '@mui/icons-material';
+import { fetchUsers } from '../userSlice';
+// import { fetchUsers, getUsers } from '../userSlice';
```

Comment:
You can remove the unnecessary comments.

Reply:
@otsembo I have made the changes. You can have a look